web/src/app/registration/registration.component.html

```diff
@@ -1,17 +1,19 @@
-<div id="message"></div>
-<form action="">
-    <label for="firstname">First name:</label>
-    <input type="text" name="firstname" id="firstname">
-    <br>
-    <label for="lastname">Last name:</label>
-    <input type="text" name="lastname" id="lastname">
-    <br>
-    <label for="email">Email:</label>
-    <input type="email" name="email" id="email">
-    <br>
-    <label for="password">Password:</label>
-    <input type="password" name="password" id="password">
-    <br>
-    <input id="register" type="button" value="Register">
-</form>
-<div>Already have an account? Log in</div>
\ No newline at end of file
+<div id="formContainer">
+    <div id="message"></div>
+    <form action="">
+        <label for="firstname">First name:</label>
+        <input type="text" name="firstname" id="firstname">
+        <br>
```

Comment:
Instead of using `<br>`, it would be better to wrap each `label` and `input` with a `div`, e.g.:
```
<div class="form-group">
      <label for="firstname">First name:</label>
      <input type="text" name="firstname" id="firstname" />
</div>
```
and apply `flex-direction: column` to this div. `margin-bottom` can also be added to create spacing between input fields, e.g.:

```
.form-group {
      display: flex;
      flex-direction: column;
      margin-bottom: 10px;

      label {
        margin-bottom: 10px;
          ...
      }

      input {
          ...
      }
}
```

Reply:
Removed br elements and added div wrapping to label & input pairs.